java/java.lsp.server/nbcode/integration/nbproject/project.xml

```diff
@@ -76,13 +93,39 @@
                         <specification-version>1.7.3</specification-version>
                     </run-dependency>
                 </dependency>
+                <dependency>
+                    <code-name-base>org.netbeans.modules.cloud.oracle</code-name-base>
+                    <build-prerequisite/>
+                    <compile-dependency/>
+                    <run-dependency>
+                        <implementation-version/>
+                    </run-dependency>
+                </dependency>
                 <dependency>
                     <code-name-base>org.netbeans.modules.java.lsp.server</code-name-base>
                     <build-prerequisite/>
                     <compile-dependency/>
                     <run-dependency>
                         <release-version>2</release-version>
-                        <specification-version>2.0</specification-version>
+                        <implementation-version/>
```

Comment:
Instead of introducing new implementation dependencies, better longer term solution would be to crate `CommandProvider` interface inside the `api.lsp` module and implement the `ProjectAuditCommand` directly as a part of `cloud.oracle` module

Reply:
Understood: https://issues.apache.org/jira/browse/NETBEANS-6518. I would like to introduce some (de)serialization basics, so that all modules do not need to use Jackson or GSON for trivial data structures.